bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java

```diff
@@ -39,10 +45,12 @@
 import org.eclipse.jdt.annotation.Nullable;
 import org.jellyfin.sdk.api.client.exception.ApiClientException;
 import org.jellyfin.sdk.model.api.BaseItemDto;
+import org.jellyfin.sdk.model.api.BaseItemKind;
 import org.jellyfin.sdk.model.api.PlayCommand;
 import org.jellyfin.sdk.model.api.PlayerStateInfo;
 import org.jellyfin.sdk.model.api.PlaystateCommand;
 import org.jellyfin.sdk.model.api.SessionInfo;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Strange import, to be checked.
Your intention was probably rather to use JDT annotation.